rdr_service/resource/generators/participant.py

```diff
@@ -831,8 +821,9 @@ def _prep_physical_measurements(self, p_id, ro_session):
         results = query.all()
 
         for row in results:
-
-            if row.final == 1 and row.status != PhysicalMeasurementsStatus.CANCELLED:
+            # row.final is not a complete indicator of a finalized PM record, there are cases where row.final = 0,
```

Comment:
FYI for @wangy70:  Robert and I were discussing this and this logic will be superceded with an upcoming PR to more closely align with the RDR logic for assigning `physical_measurements_status` field in `participant_summary`